calico-enterprise/networking/egress/egress-gateway-aws.mdx

```diff
@@ -407,6 +412,7 @@ kubectl patch felixconfiguration default --type='merge' -p \
 
 - `egressIPSupport` must be the same on all cluster nodes, so you should set them only in the
   `default` FelixConfiguration resource.
+- As of v3.16.1, the operator will automatically enable the required policy sync API in the FelixConfiguration.
```

Comment:
```suggestion
- The operator automatically enables the required policy sync API in the FelixConfiguration.
```
Best not to talk about specific version numbers, particularly when we're talking about future capabilities. If this is landing in 3.16.1, we can consider leaving this out for now and adding that information along with the release.